pkg/daemon/update.go

```diff
@@ -46,6 +46,9 @@ const (
 	fipsFile              = "/proc/sys/crypto/fips_enabled"
 	extensionsRepo        = "/etc/yum.repos.d/coreos-extensions.repo"
 	osImageContentBaseDir = "/run/mco-machine-os-content/"
+	cliBinPath            = "/usr/bin/oc"
+	cliRunBinPath         = "/run/bin/oc"
```

Comment:
Though something that will become important with this is that `/run/bin` isn't in `$PATH`.  We can fix that pretty easily but if we take this approach out farther we'll likely want to do something more sophisticated.